packages/storybook/stories/SendBox/SendBox.stories.tsx

```diff
@@ -98,11 +98,17 @@ const getDocs: () => JSX.Element = () => {
 };
 
 const SendBoxStory = (args): JSX.Element => {
+  const timeoutRef = React.useRef<any>();
+  const delayForSendButton = 300;
   return (
     <div style={{ width: '31.25rem' }}>
       <SendBoxComponent
         disabled={args.disabled}
-        onSendMessage={async (message) => alert(`sent message: ${message} `)}
+        onSendMessage={async (message) => {
+          timeoutRef.current = setTimeout(() => {
```

Comment:
Probably not very valid situation, but we may need to cancel timer in case if another one is created